<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="gbItems">
        <property name="title">
         <string>Items</string>
        </property>
<widget class="QRadioButton" name="rbCircle">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Circle</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbTriangle">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Triangle</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbSquare">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Square</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbRectangle">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Rectangle</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbPolygon">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Polygon</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
